cachi2/core/package_managers/npm.py

```diff
@@ -539,4 +673,5 @@ def _resolve_npm(pkg_path: RootedPath) -> ResolvedNpmPackage:
         "dependencies": package_lock.get_sbom_components(),
         "package_lock_file": package_lock.get_project_file(),
         "dependencies_to_download": package_lock.get_dependencies_to_download(),
+        "workspaces": package_lock.workspaces,
```

Comment:
In general, I think all the replacing can happen in resolve_npm instead of returning helper info from resolve_npm and having fetch_npm_source do the work

Reply:
True, discussed. Logic will move to `resolve_npm`.